### DISCUSSION TITLE
mapping ports to the image & model

### DISCUSSION THREAD

**Discussion by georgedavid29** [netbox-community/devicetype-library#1759] [Ideas]
it would be amazing if we can map the ports and interfaces to the front and back images. we often have remote hands go out to service devices and it would be great to be able to send a screenshot of more specifically what they need to go check.
also when a port is cabled it could show you exactly where on the device its connected to

**Comment by danner26** [comment]
This currently is not possible here, but if you'd like you would need to submit this as an idea in the upstream NetBox repo. Please start by checking if it has already been proposed in the discussions/issues before creating a new discussion
